ballerina/xml_api.bal

```diff
@@ -68,9 +68,11 @@ public type Options record {|
 # Represent the options that can be used to modify the behaviour of projection.
 #
 # + allowDataProjection - enable or disable projection
+# + isSemantic - choose between semantic and syntactic equality
 public type SourceOptions record {|
```

Comment:
Not particularly introduced in this PR, but fields should be documented at field level?